app/translations/ru.json

```diff
@@ -111,7 +111,7 @@
   "app.common.showLess": "Показать меньше {value}",
   "app.common.poweredByTelos": "Powered by {image}",
   "app.common.poweredByPeeranha": "Powered by {image}",
-  "app.common.termsOfTelos": "Telos Operating Agreement",
+  "app.common.termsOfTelos": "Operating Agreement",
```

Comment:
key should be another one, also russian translation here, not english

Reply:
changed: https://github.com/peeranha/peeranha-web/pull/650/commits/fcf50b44b1470726b3dd53763fca278224a03a17